src/isValidQuery.js

```diff
@@ -0,0 +1,6 @@
+const minMax = new RegExp(/(\()(?:[a-z][a-z]+)(-)((?:[a-z][a-z]+))?(-)?((?:[a-z][a-z]+))(:)(\s?)(\d+)((?:[a-z][a-z]+))\)/)
```

Comment:
I think there's quite a few cases that this doesn't cover - can't we rely on the matchMedia API to do validation?